server/cmd/mmctl/commands/team.go

```diff
@@ -346,17 +348,21 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
-			printer.PrintError("Unable to find team '" + args[i] + "'")
+			errText := "Unable to find team '" + args[i] + "'"
+			printer.PrintError(errText)
```

Comment:
In other parts of the code, we simply print the multierror before returning it at the end of the function. I wonder if that would make the code more readable.

Reply:
Nice. another improvement. 

I also flattened out the if else statement. Will that be better?